port.h

```diff
@@ -114,4 +115,13 @@ SOFTWARE.
 #define BRICKS_HAS_THREAD_LOCAL
 #endif
 
+// `std::make_unique` exists in C++14, but for Bricks we'd like to see it "supported" in C++11. -- D.K.
```

Comment:
True, I suggest we revisit it once C++14 starts taking over. Doesn't look important or urgent as of now.